### ISSUE TITLE
Accessibility scan

### ISSUE THREAD

**Issue by amgdev9** [mobile-next/mobilewright#235]
### Problem or motivation

I think it could be useful to call the native accessibility scan (espresso a11y checks on android and performAccessibilityAudit on iOS) so its possible to automate some accessibility aspects

### Proposed solution

Provide a method on the screen object to perform the a11y audit, same call for both platforms and returns an array of a11y violations

### Target platform

Both

**Comment by gmegidish** [comment]
@amgdev9 Yes. It's possible to use mobilewright for accessibility scan. Right now that's easily doable with mobile-mcp and appropriate skill or checklist. We'll be adding this as a mobile-mcp skill, and a mobilewright feature in the future. Thanks for the question! Are you blocked right now?
